monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -0,0 +1,149 @@
+import hashlib
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+from tests.utils import is_user_admin
+
+from infection_monkey.ransomware.ransomware_payload import RansomewarePayload
+
+EXTENSION = ".m0nk3y"
```

Comment:
Can't we just import the extension from `monkey/infection_monkey/ransomware/ransomware_payload.py`?